pkg/operator/mon/mon.go

```diff
@@ -40,6 +40,10 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mon")
 
 const (
+	// EndpointConfigMapName is the name of the configmap with mon endpoints
+	EndpointConfigMapName = "mon-config"
```

Comment:
the const name implies the CM is only for endpoints, but the value implies its more general.  Should these match better?